utils/moreutils/patches/001_disable-manuals.patch

```diff
@@ -0,0 +1,11 @@
+--- a/Makefile
```

Comment:
I would just drop this patch completely. We generally don't do this for other packages either.